.gitignore

```diff
@@ -1,3 +1,6 @@
 *pyc
-.vscode/settings.json
+.vscode/
 */doc/generated
+/build/
+/log/
+/install/
```

Comment:
these folders shouldn't be inside your checked out repo - the usual layout is:

```
build
log
install
src
   this_repo
   other_repos
```

See https://docs.ros.org/en/rolling/Tutorials/Beginner-Client-Libraries/Creating-A-Workspace/Creating-A-Workspace.html

Reply:
Agreed i will remove the changes in the gitigniore.